docs/learning/transformer/getting-started.md

```diff
@@ -0,0 +1,87 @@
+# Getting Started with iModel-Transformer
+
+## Overview
+
+The repository supports both JavaScript and TypeScript. To build a strong foundation, check out these resources:
+
+- **JavaScript:**  
+  [The Modern JavaScript Tutorial](https://javascript.info/)  
+  [Mozilla JavaScript Reference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference)
+  
+- **TypeScript:**  
+  [TypeScript Documentation](https://www.typescriptlang.org/docs/)
+
+For more details on the transformer itself:
+
+- **Transformer Repo:** [GitHub - imodel-transformer](https://github.com/iTwin/imodel-transformer)
+- **Transformer Docs:** [iTwinJS Transformer Learning](https://www.itwinjs.org/learning/transformer)
+- **Branching & Synchronization:** [Branching iModels Documentation](https://github.com/iTwin/imodel-transformer/blob/main/docs/learning/transformer/branching-imodels.md)
+
+## iModel Files, Changesets & Checkpoints
+
+Below is an image of an iModel Checkpoint showcasing some important concepts that are good to know:
+
+![IModel Checkpoint](./iModelCheckpoint.png)
+
+- **Seed File (S):**  
+  This is the base state of an iModel.
+
+- **Changesets:**  
+- A Changeset represents a group of changes to an iModel. Changesets are created whenever a local copy of the iModel is modified and reflect the union of all additions, deletions, and modifications over a period of time.
+  Incremental updates to the iModel. For example:
+  - To reach **Point A**, download the seed file (S) and apply changesets 1, 2, and 3 in order.
+  - To reach **Point B**, download the seed file (S) and apply changesets 1 through 6 sequentially.
+
+- **Checkpoints:**  
+  A checkpoint is a full snapshot of the iModel at a given point, with changesets already applied. This can be used to reduce number of Changesets needed to apply to get to a certain version of the iModel.
+  - **Checkpoint X** might represent the state after changesets 1–3, letting you quickly access **Point A**.
+  - To reach **Point B**, you might download **Checkpoint Y** and apply only the remaining changes (e.g., changeset 6).
+
+## Key Concepts
+
+- **Snapshot:**  
+  A [static](https://www.itwinjs.org/learning/backend/accessingimodels/#snapshot-imodels) point-in-time representation of an iModel’s state.
```

Comment:
See comment above about relative links and using $docs. There are several more instances in this document as well.